File: backend/Testing/Browser/ManagerPageTest.cs
using Testing.Browser.Base;
using Testing.Browser.Page;
using Testing.Services;

namespace Testing.Browser;

[Trait("Category", "Integration")]
public class ManagerPageTest : PageTest
{
    [Fact]
    public async Task DisplayDashboardThenProjectPage()
    {
        var loginPage = await new LoginPage(Page).Goto();
        await loginPage.FillForm("manager", TestingEnvironmentVariables.DefaultPassword);

        await loginPage.Submit();
        var userDashboardPage = await new UserDashboardPage(Page).WaitFor();

        await userDashboardPage.OpenProject("sena 3", "sena-3");
        await Expect(Page.GetByText("Project Code: sena-3")).ToBeVisibleAsync();
    }
}

File: frontend/tests/managerPage.test.ts
import { expect, test } from '@playwright/test';
import { LoginPage } from './pages/loginPage';
import { defaultPassword } from './envVars';
import { UserDashboardPage } from './pages/userDashboardPage';

test('display dashboard, then project page', async ({ page }) => {
  const loginPage = await new LoginPage(page).goto();
  await loginPage.fillForm('manager', defaultPassword);
  await loginPage.submit();
  const userDashboardPage = await new UserDashboardPage(page).waitFor();
  await userDashboardPage.openProject('Sena 3', 'sena-3');
  await expect(page.getByText('Project Code: sena-3')).toBeVisible();
});
